framework/include/materials/ThreeMaterialPropertyInterface.h

```diff
@@ -0,0 +1,100 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "TwoMaterialPropertyInterface.h"
+
+// Forward Declarations
+class MaterialData;
+class ThreeMaterialPropertyInterface : public TwoMaterialPropertyInterface
```

Comment:
docstring

what is this for? Adding face materials to element and neighbor?

I dont like the name either. 
What about (Inter)faceMaterialPropertyInterface

Reply:
I will add a docstring. I think the name is aligned with our naming, otherwise we should change the name of `TwoMaterialPropertyInterface`. I personally think `ThreeMaterialPropertyInterface` is good and intuitive. `InterfaceMaterialPropertyInterface` is not accurate unless name it like `DomainMaterialPropertyInterface` but I think it not as simple as the name here.